modules/mod_filestore/templates/admin_filestore.tpl

```diff
@@ -56,11 +56,26 @@
                                 <div class="checkbox col-md-12">
                                     <label for="is_upload_enabled">
                                         <input type="checkbox" id="is_upload_enabled" name="is_upload_enabled" {% if m.config.mod_filestore.is_upload_enabled.value == "true" %}checked{% endif %} />
-                                        {_ Upload new media files to the cloud by default _}
+                                        {_ Upload new media files to the cloud file store _}
                                     </label>
                                 </div>
                             </div>
 
+                            <div class="form-group row">
+                                <div class="col-md-12">
+                                    <label for="delete_interval" class="control-label">{_ Delete files from the cloud file store _}</label>
+                                    {% with m.config.mod_filestore.delete_interval.value|default:"0" as value %}
+                                        <select class="form-control-inline" id="delete_interval" name="delete_interval">
+                                            <option value="false"{% if value == false %} selected{% endif %}>{_ Never _}</option>
```

Comment:
I would put Never as last option, so you get a gradual increase from now to never.